projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item-component.model.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { OrderEntry, PromotionLocation } from '@spartacus/core';
+import { ReplaySubject } from 'rxjs';
+import { CartItemComponentOptions } from '../cart-item-list/cart-item-list.component';
+
+export enum CartItemComponentOutlets {
+  CONFIGURATOR = 'cx-cart-item.configurator',
```

Comment:
Here I would propose generic names without relation to the configurator. I have them as item.start and item.information in my implementation